src/main.tsx

```diff
@@ -2,17 +2,19 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App'
 import './App.css'
-import Login from './components/Login'
-import CheckIn from './components/CheckInTable'
 import './index.css'
 import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
-import { Typography } from '@mui/material'
+import { GoogleOAuthProvider } from '@react-oauth/google'
+
+const clientId = '8990298927-tftcl4kcfe52ikmu3quthlo7v3qvognu.apps.googleusercontent.com';
```

Comment:
No action needed, but I don't like clientId being static here. We'll probably introduce an env variable for it eventually.

Reply:
That makes sense. This one was just for testing and can be changed to an env once we are ready for production